tests/containers/bci_logs.pm

```diff
@@ -47,6 +50,15 @@ sub run {
                 # Replace default attribute name "pytest" by its env name and engine
                 my $new_name = $env . '_' . $engine;
                 $node->{name} =~ s/pytest/$new_name/;
+                for my $case ($node->findnodes('//testcase')) {
+                    for my $skipped ($case->findnodes('//skipped')) {
```

Comment:
The skipped ones don't matter, they should not be softfailed.

We should look for the softfailed entries, or xfail(ed) as they are in the pytest environment.